generic3g/specs/UnitsAspect.F90

```diff
@@ -14,7 +14,7 @@ module mapl3g_UnitsAspect
 
 
    type, extends(StateItemAspect) :: UnitsAspect
-      private
```

Comment:
Do we really want a public UnitsAspect?

Reply:
For now yes.     I will likely add accessors when the dust settles, but inserting getters everywhere that units are currently referenced is counter productive.  Most will not be needed in the end.